src/group.rs

```diff
@@ -157,22 +162,35 @@ impl GroupOps for crate::IronOxide {
         rt.block_on(group_api::list(self.device.auth(), None))
     }
 
-    fn group_create(&self, opts: &GroupCreateOpts) -> Result<GroupMetaResult> {
+    fn group_create(&self, opts: &GroupCreateOpts) -> Result<GroupCreateResult> {
         let mut rt = Runtime::new().unwrap();
         let GroupCreateOpts {
             id: maybe_id,
             name: maybe_name,
             add_as_member,
+            members,
             needs_rotation,
         } = opts.clone();
 
+        let modified_add_as_member;
+        if members.contains(self.device.auth().account_id()) {
+            modified_add_as_member = true;
+        } else {
+            modified_add_as_member = add_as_member;
+        }
+        let modified_members: Vec<UserId> = members
+            .into_iter()
+            .filter(|id| id.id() != self.device.auth().account_id().id())
```

Comment:
`.filter(|id| id != self.device.auth().account_id())` -- You can compare using the id like this instead.

Reply:
I've changed this section, but it's good to know that I can compare the UserIds directly